website/docs/d/portgroup.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_portgroup"
+sidebar_current: "docs-vcd-data-source-portgroup"
+description: |-
+  Provides a data source for available vCenter Port Groups.
+---
+
+# vcd\_portgroup
+
+Provides a data source for available vCenter Port Groups.
+
+Supported in provider *v3.0+*
```

Comment:
VCD version missing

Reply:
This works even with 9.7 so not sure we need it.